Orange/widgets/visualize/owmosaic.py

```diff
@@ -383,19 +382,34 @@ def init_combos(self, data):
             if attr.is_discrete or attr.is_continuous:
                 for combo in self.attr_combos:
                     combo.addItem(icons[attr], attr.name)
+        self.cb_attr_color.addItem("(Pearson residuals)")
+        for attr in chain(data.domain.class_vars, data.domain.attributes, data.domain.metas):
```

Comment:
IIRC we wanted to transition to a new order in dropdowns of: class_vars, metas, attributes?
Since some data can have thousands of attributes and scrolling to the end can be a pain.
To be double-checked.

Even better - use DomainModel. I think it should be possible to add a custom string at the beginning.